provisioner/plain-v0/README.md

```diff
@@ -96,6 +96,8 @@ NAME                 DESIRED BUNDLE   INSTALLED BUNDLE   INSTALL STATE
 my-bundle-instance   my-bundle        my-bundle          InstallationSucceeded   11s
 ```
 
+> Note: Creation of BundleInstances from the same Bundle will result in an error if any namespaced resources (CRDs, Namespaces, etc) are defined in the Bundle that both BundleInstances reference. 
```

Comment:
A couple of things:

- Can we move this note to be above the command output?
- CRDs are cluster-scoped resources
- Should we just document that 2+ BundleInstances referencing the same Bundle will result in an error (or it's unsupported workflow)?

Reply:
🤦🏻 I meant to say cluster-scoped here, not name-spaced scoped - my bad. When you say document do you mean run through an example of this like we do for the other use cases?

Reply:
Nope - s/document/make a note. So basically, instead of specifying what scenarios two BIs targeting the same Bundle is problematic (e.g. it contains cluster-scoped resources), we just make a note that this workflow is unsupported, and you'll likely get an error. Thoughts?

Reply:
Ah okay, so moving from partial support to entirely not supporting it. I agree with that and will make the update.